test/cli/test-inline-suppress.py

```diff
@@ -19,3 +19,8 @@ def test_unmatched_suppression():
     ret, stdout, stderr = cppcheck(['--inline-suppr', '--enable=information', '--error-exitcode=1', 'proj-inline-suppress/2.c'])
     assert ret == 1
     assert 'Unmatched suppression: some_warning_id' in stderr
+
+def test_unmatched_suppression_2():
```

Comment:
I suggest the name `test_simplify_path` or `test_path_with_extra_stuff` or something like that.

Reply:
or even `test_unmatched_suppression_path_with_extra_stuff` it seems like a good idea that it contains `unmatched_suppression`.